src/components/include/cos_component.h

```diff
@@ -23,7 +23,7 @@ int call_cap_asm(u32_t cap_no, u32_t op, int arg1, int arg2, int arg3, int arg4)
 	cap_no = (cap_no + 1) << COS_CAPABILITY_OFFSET;
 	cap_no += op;
 
-	__asm__ __volatile__("":::"ecx", "edx");
+//	__asm__ __volatile__("":::"ecx", "edx");
```

Comment:
Thanks, that was annoying me, and I didn't know if you had it there for a reason.